code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -147,6 +150,9 @@ namespace <#= qualifiedNamespace #>
                 else if (!view.HasComponent<ComponentAdded<<#= componentDetails.TypeName #>>>(entity))
                 {
                     view.AddComponent(entity, new ComponentAdded<<#= componentDetails.TypeName #>>());
+<# if(fieldDetailsList.Count > 0) { #>
+                    view.AddComponentsUpdated(entity, <#= componentDetails.CamelCaseTypeName#>Update, UpdatesPool);
```

Comment:
If authority is also added this will appear as two component updates with the same data.

Reply:
Is this true? @martinzlocha Could we get a full file for the generated code?

Reply:
Yup, thats true, gonna write a possible solution to avoid this.

Reply:
I have updated the codegen so that it does not generate duplicate updates.